docs/source/lts_selection.org

```diff
@@ -0,0 +1,151 @@
+#+TITLE: On selecting versions as LTS candidates
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 12-08-2023
+* About
+
+In the interest of preserving the engineering resources, it is in our
+interests to limit the scope of post-release support. If one has
+several versions of Iroha running in parallel, they might need to
+replicate the support work as many times as there are parallel
+versions.
+
+Instead of attempting to support multiple versions badly, we try to
+cater to 80% of use cases with 20% of the effort and reach a
+compromise between supporting every released version equally and only
+supporting the latest stable version.
+
+Specifically, we support the latest =stable= release. Often this
+release is either not as well-tested, well-documented, or
+well-supported by the SDKs, in addition to often changing in terms of
+its API and ABI, resulting in a moving target for compatibility. For
+this reason we designate a specific version that is at least one of 12
+releases to be the Long-term supported release following the criteria
+listed below.
+
+* LTS selection criteria
+
+The following is a non-exhaustive list of recommended considerations
+for selecting a particular version as an LTS candidate
+
+1. The release should be /unlikely/ to contain major /known/ issues which
+   would preclude it from being useful.
+
+   More specifically, the release should have a good coverage of a
+   basic use-case, for example, RC-6 was the last release which
```

Comment:
I'd also suggest elaborating a bit on 'RC' labelling itself (agreed ~monthly build etc) or placing a reference if that's described somewhere else

Reply:
I am confused by RC (release candidate) labelling in general. In fact, none of the releases were "release candidates", but more likely alpha- or beta- releases. IMO, release candidate implies feature freeze.

Also, we actually don't have RCs, but _pre_ RCs: `2.0.0-pre-rc.6`